src/bin/compute-gop.cc

```diff
@@ -63,7 +63,17 @@
 
 namespace kaldi {
 
-/** FrameLevelLpp compute a log posterior for pure-phones by sum the posterior
+int32 PhoneNum(const std::vector<std::set<int32> > &pdf2phones) {
```

Comment:
It's a small issue, just noted we can do this faster, as `std::set` is sorted, we can get phone_num just as below:

```
for(auto &pdf: pdf2phones){
if(!pdf.empty()) phone_num = std::max(phone_num, 1+ pdf.rbegin());
}
```

Reply:
Thanks haowen, I didn't know `std::set` is sorted. @wangyongqing0731 has modified it as your suggestion, but I think it should be `1+ *pdf.rbegin()` rather than `1+ pdf.rbegin()`. :sweat_smile: